contract/views/contract.xml

```diff
@@ -288,21 +310,17 @@
                             </field>
                         </page>
                         <page name="info" string="Other Information">
-                            <field name="create_invoice_visibility" invisible="1" />
                             <group>
                                 <field name="code" />
                                 <field name="group_id" />
-                                <field
-                                    name="company_id"
-                                    options="{'no_create': True}"
-                                    groups="base.group_multi_company"
-                                />
                                 <field
                                     name="currency_id"
                                     groups="base.group_multi_currency"
                                 />
-                                <field name="invoice_partner_id" required="1" />
                             </group>
+
+                            <field name="create_invoice_visibility" invisible="1" />
+
```

Comment:
Why is this field needed? In v18 no need to add it in the view if it is only for an invisible/readonly domain.